index.js

```diff
@@ -316,6 +314,23 @@ function parseAdapterFlags(adapter) {
 const requestLink = 'https://webgpufundamentals.org/webgpu/lessons/webgpu-limits-and-features.html';
 const requestHint = 'limits greater than default must be specified when requesting adapter';
 
+function adapterWarnings(adapter, device) {
+  const rows = [];
+
+  const addRow = (msg) => {
+    if (rows.length === 0) {
+      rows.push(el('tr', { className: 'warn' }, [el('td', {colSpan: 2, textContent: 'warnings:'})]));
+
+    }
+    rows.push(el('tr', { className: 'feature warn' }, [el('td', {colSpan: 2, textContent: msg}), el('td')]));
+  }
+
+  if (!device.importExternalTexture) {
```

Comment:
Why not testing `'importExternalTexture' in GPUDevice.prototype` directly instead?